### PR TITLE
Explain the impacts of the phase unwrapping reference point on measurements

### PR DESCRIPTION
**Pull request by hjkristenson**
This is the first round of additions to clarify the impact of the phase unwrapping reference point on interferogram values. 

It addresses the fundamental impacts the choice of reference point location has on the phase difference (unwrapped interferogram) and LOS displacement values.

It can be expanded further as we understand the products better, specifically how GAMMA assigns a reference point if the 0,0 location is a masked pixel (poor coherence, low amplitude, or water masked). 

Another round of information can be added addressing the vertical displacement maps in particular once we have a better handle on how GAMMA produces those maps and how they should be interpreted.

### CODE REVIEW

File: docs/guides/insar_product_guide.md [hunk 1]
```diff
@@ -193,6 +193,11 @@ Coherence is estimated from the normalized interferogram and the co-registered i
 
 When the water masking option is applied, the validity mask is further amended to apply 0 values to any pixels classified as water in the water mask. In some cases, pixels over water may still meet the coherence and amplitude threshold criteria for inclusion, even though they are not valid for use during phase unwrapping. When processing scenes with extensive coverage by coastal waters or large inland waterbodies, there can be erroneous phase jumps introduced if unwrapping proceeds over water as if it were land. In such cases, choosing the option to apply the water mask can significantly improve the results. 
 
+#### Reference point
+In order to perform phase unwrapping, a reference point must be selected. The unwrapping will proceed relative to this reference point; the 2&#960 integer multiples will be applied to the wrapped phase using this pixel as the starting point.
```
Comment (jhkennedy) [inline docs/guides/insar_product_guide.md:197]: You should be able to directly use the [unicode π](https://unicode-table.com/en/03C0/) here instead of the HTML code. While the HTML code should render correctly on the web, it doesn't render in GitHub:

pi

**Review by jhkennedy — APPROVED** [review]
I like this! One small, non-blocking comment is all.

File: docs/guides/insar_product_guide.md (hunk 1 above)
Comment (hjkristenson) [reply docs/guides/insar_product_guide.md]: @jhkennedy Can you make that magic happen? I tried just replacing the html pis with unicode pis, but clearly I did something wrong. I was just re-using the pi symbol that had been used by others in the rest of the document.

Comment (hjkristenson) [reply docs/guides/insar_product_guide.md]: @jhkennedy Nevermind! I think I got it corrected.

Comment (jhkennedy) [reply docs/guides/insar_product_guide.md]: Yep looks great!

**Review by jhkennedy — APPROVED** [review]

File: docs/guides/insar_product_guide.md [hunk 2]
```diff
@@ -344,6 +349,15 @@ When looking at a single interferogram, the only reliable deformation measuremen
...
+The reference point for phase unwrapping is set to be the upper left corner of the image by default. This may not always be an ideal location. If it's in an area of low coherence, or if it's in a patch of coherent pixels that is separated from the area undergoing deformation by a gap of incoherent pixels, the unwrapping may be of lower quality than if the reference point was in a more suitable location. 
+
+Even when there are not phase unwrapping errors introduced by phase discontinuities, it is important to be aware that unwrapped phase differences and displacement values are all relative to the reference point. If you are interested in the absolute change in displacement in a particular area, you may want to apply a correction to the image so that the values you are working with are in terms that are more intuitive for your study area or application.
+
+In cases where there are not significant phase discontinuities, you can select a reference point that is optimal for your use case, calculate the difference in phase or line-of-sight displacement, and apply a correction to the entire raster to adjust the values to be relative to that new reference point.
```
Comment (talogan) [inline docs/guides/insar_product_guide.md:357]: I think this needs more clarification - perhaps an example, like 

_If the reference point at 0,0 has a phase of 0.3, and your desired reference point has a phase of 0.7, you would subtract 0.7 from all of the phase values in your image, changing the initial reference point to a value of -.4._

**Review by talogan — APPROVED** [review]

File: docs/guides/insar_product_guide.md (hunk 2 above)
Comment (hjkristenson) [reply docs/guides/insar_product_guide.md:357]: I added an example box for this [in this commit](https://github.com/ASFHyP3/hyp3-docs/pull/158/commits/fd1a78bf9126c7201633b64c3d145380340c3ad2). Might be too wordy. Thought I might add an image to help illustrate. Will follow up next week. Comments welcome.

Comment (hjkristenson) [reply docs/guides/insar_product_guide.md]: The story was a bit complicated when adding images. At @talogan's suggestion, I just took the first sentence from the example box and added it to the previous paragraph (see [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/158/commits/c86484d957abfd24bc0d05698f2a4ea525ac2935). Someday I hope to put together a story map that dives into this more.
